pennylane/ops/qubit/parametric_ops.py

```diff
@@ -810,7 +810,7 @@ class PauliRot(Operation):
     }
 
     def __init__(self, theta, pauli_word, wires=None, do_queue=True):
-        super().__init__(theta, pauli_word, wires=wires, do_queue=do_queue)
```

Comment:
Sorry for sneaking this in too, but I needed a nice gate that defines everything, and PauliRot was perfect - but it was not yet updated to store hyperparams correctly. So I quickly did that